cypress/support/api/circulation.js

```diff
@@ -29,7 +29,12 @@ Cypress.Commands.add('createItemCheckinApi', (data) => {
       id: uuid(),
```

Comment:
try to move createItemCheckinApi into related fragment and stop to use cy.env. Work through return value please

Reply:
You suggest creating a separate method for callback of `after()` block. But as I see from the code perspective we do not use that approach and directly write the code in `after()` block

Reply:
let's discuss this in call by voice

Reply:
done

Reply:
It has related fragments. 
Or do you mean that we should have separated fragments that contain ONLY api calls?

Reply:
"It has related fragments." - could you move these methods into related existing fragements?
"do you mean that we should have separated fragments that contain ONLY api calls?" - no. Idea - bisuness related with GUI fragments-> fragments has related with business GUI and api methods

Reply:
But I already moved `createItemCheckinApi` to already existing `fragments/check-in-actions/checkInActions.js` file.
And `checkoutItemApi` to `fragments/checkout/checkout.js` file.

Reply:
let's remove cypress/support/api/circulation.js in this case

Reply:
Done